### ISSUE TITLE
Add Support for apps.manifest.* Endpoints

### ISSUE THREAD

**Issue by misscoded** [slackapi/java-slack-sdk#848]
Add support for the following `apps.manifest.*` methods:

`apps.manifest.create`
`apps.manifest.validate`
`apps.manifest.update`
`apps.manifest.delete`
`apps.manifest.export`

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [x] **slack-api-client** (Slack API Clients)
* [ ] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

### Requirements

Please make sure if this topic is specific to this SDK. For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to the those rules.

**Comment by cts5ws** [comment]
Is this still being prioritized?

**Comment by seratch** [comment]
@cts5ws Yes! but we're planning to add the support when those APIs are GAed.

**Comment by zimeg** [comment]
🔮 Let's close this issue with https://github.com/slackapi/java-slack-sdk/pull/1231 of release [`v1.36.0`](https://github.com/slackapi/java-slack-sdk/releases/tag/v1.36.0)!
